.github/workflows/drs_ci.yml

```diff
@@ -59,12 +59,12 @@ jobs:
       ORCA_JOB: ISOLATED_TEST_ON_CURRENT
       CI: TRUE
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php-version }}
           coverage: xdebug
-          ini-file: development
+          ini-values: assert.active=1, assert.exception=1, zend.assertions=1
```

Comment:
Do we have to set this explicitly?
![image](https://github.com/acquia/drupal-recommended-settings/assets/63041696/be865026-56fc-4502-bd7b-3c3db058b3de)

Reply:
No much required, but advisable to set this, as community follows the same. Reference: https://github.com/php-tuf/composer-stager/blob/develop/.github/workflows/main.yml#L59-L62